pkg/alerts/nrql_conditions.go

```diff
@@ -6,6 +6,42 @@ import (
 	"github.com/newrelic/newrelic-client-go/pkg/errors"
 )
 
+// AlertsNrqlConditionExpiration - **Preview access:** These fields may be viewed and set, but will not be active until the release date.
+// Settings for how violations are opened or closed when a signal expires.
+// nolint:golint
+type AlertsNrqlConditionExpiration struct {
+	ExpirationDuration          *int `json:"expirationDuration"`
```

Comment:
`omitempty`?

Reply:
Same question for the other pointers in `AlertsNrqlConditionSignal`.

Reply:
The nuance of the GraphQL service, merging of previous/new fields, optional values, default values in the API, and backwards compatible fields requires a somewhat specific combination of pointers and inclusion/exclusion of `omitempty`.

For the most part, these have to be pointers in order to allow the Terraform to be backwards compatible (not requiring users to begin setting these options until they're ready). GraphQL differentiates between nil with `omitempty` (Javascript `undefined`) versus nil without `omitempty` (Javascript `null`). In order to support options like `fillValue`, it is required some fields must have the ability to be null.